tests/testthat/test-exporters.R

```diff
@@ -98,13 +98,14 @@ test_that("exporters work", {
         mymf <- MatrixPrintForm(strings = strs, aligns = matrix("center", ncol = 2, nrow = 5),
                                 formats = matrix("xx", ncol = 2, nrow = 5),
                                 spans = matrix(1L, ncol = 2, nrow = 5),
+                                has_topleft = FALSE,
                                 line_grouping = c(1, 1, 2, 3, 4),
```

Comment:
not having to add topleft info was a problem as it was mistakingly considering the first colname as a topleft info